modules/thermo-cycler/thermo-cycler-arduino/lid.h

```diff
@@ -40,6 +40,13 @@
 #define STEPS_PER_MM 15  // full-stepping
 #define PULSE_HIGH_MICROSECONDS 2
 
+typedef enum{
+  UNKNOWN = 0,
+  CLOSED,
+  OPEN
+}Lid_status;
+
+static const char * LID_STATUS_STRINGS[] = {"Unknown", "Closed", "Open"};
```

Comment:
This shouldn't be in the header, it will cause it to be defined in every translation unit. This should either be declared here and defined in the file or declared and defined as a `static const` public member of the `Lid` class.